src/main/bin/dita.bat

```diff
@@ -66,7 +67,7 @@ if "%_JAVACMD%" == "" set _JAVACMD=java.exe
 
 :runAnt
 rem sun.io.useCanonCaches improves performance in Java 12+ (see https://bugs.openjdk.java.net/browse/JDK-8207005)
-"%_JAVACMD%" %ANT_OPTS% -Djava.awt.headless=true -Dcli.log-format=fancy -Dcli.color=true -Dsun.io.useCanonCaches=true -classpath "%DITA_HOME%\lib\ant-launcher.jar;%DITA_HOME%\config" "-Dant.home=%DITA_HOME%"  "-Ddita.dir=%DITA_HOME%" org.apache.tools.ant.launch.Launcher %ANT_ARGS% -cp "%CLASSPATH%" %DITA_CMD_LINE_ARGS% -buildfile "%DITA_HOME%\build.xml" -main "org.dita.dost.invoker.Main"
+"%_JAVACMD%" -Djava.awt.headless=true -Dcli.log-format=fancy -Dcli.color=true -Dsun.io.useCanonCaches=true %ANT_OPTS% -classpath "%DITA_HOME%\lib\ant-launcher.jar;%DITA_HOME%\config" "-Dant.home=%DITA_HOME%"  "-Ddita.dir=%DITA_HOME%" org.apache.tools.ant.launch.Launcher %ANT_ARGS% -cp "%CLASSPATH%" %DITA_CMD_LINE_ARGS% -buildfile "%DITA_HOME%\build.xml" -main "org.dita.dost.invoker.Main"
```

Comment:
> Move ANT_OPTS so that it can be use to override cli parameters

@jelovirt & @jlacour31 Missed this before merge, but shouldn't we sync this change to the placement of `ANT_OPTS` to `bin/dita` as well? 🤔

Reply:
@jelovirt I still have the branch on my fork, can I commit the modification suggested by @infotexture? Could you re-merge it? Or should I create another PR?

Reply:
@jelovirt / @infotexture I've created https://github.com/dita-ot/dita-ot/pull/4551